composer.json

```diff
@@ -140,7 +140,7 @@
         "drupal/exif_orientation": "^1.0",
         "drupal/features": "3.8",
         "drupal/field_group": "3.1",
-        "drupal/flag": "4.0-alpha3",
+        "drupal/flag": "4.0-beta2",
```

Comment:
Is this necessary for this feature? If yes, please add it the problem and solution section of the description.

Reply:
I forget to revert it back since we don't need it until we will use a select exposed filter for Mute/Unmute.
So I reverted the version back for now.